django/contrib/auth/base_user.py

```diff
@@ -52,19 +52,17 @@ class AbstractBaseUser(models.Model):
 
     REQUIRED_FIELDS = []
 
+    # Stores the raw password if set_password() is called so that it can
+    # be passed to password_changed() after the model is saved.
+    _password = None
```

Comment:
Is this safe? Wouldn't `self._password = x` inside a method update the *class* `_password` attribute, and thus all instances of it?

Reply:
> Is this safe? 
Yes.

Reply:
If I understand correctly, `AbstractBaseUser._password = x` could update all instances, however, that doesn't seem like code a developer would write.

Reply:
You're quite right, sorry, not sure how I came up with that idea.